From 0b7086494e9fe056f35560bcab53ff9eb0e68df4 Mon Sep 17 00:00:00 2001
From: Mark Stapp <mjs@voltanet.io>
Date: Wed, 19 Feb 2020 08:44:05 -0500
Subject: [PATCH] zebra: fix missing route-advert stubs

Stubs are used when frr is built without route-advert
support; a couple of apis were missing, causing builds to fail.

Signed-off-by: Mark Stapp <mjs@voltanet.io>
---
 zebra/rtadv.c | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/zebra/rtadv.c b/zebra/rtadv.c
index e9a97d4b15..c710978d78 100644
--- a/zebra/rtadv.c
+++ b/zebra/rtadv.c
@@ -2399,4 +2399,15 @@ void rtadv_cmd_init(void)
 {
 	/* Empty.*/;
 }
+
+void rtadv_stop_ra(struct interface *ifp)
+{
+	/* Empty.*/;
+}
+
+void rtadv_stop_ra_all(void)
+{
+	/* Empty.*/;
+}
+
 #endif /* HAVE_RTADV */
